src/app/collective-rewards/leaderboard/BuildersLeaderBoardTable.tsx

```diff
@@ -0,0 +1,423 @@
+          </Typography>
+        </Popover>
+      </div>
+    </TableCell>
+  )
+}
+
+export const BackerRewardsPercentage = ({
+  rewardPercentage,
+}: {
+  rewardPercentage: BuilderRewardPercentage | null
+}) => {
+  const renderDelta = useMemo(() => {
+    if (!rewardPercentage) return null
+
+    const deltaPercentage = rewardPercentage.next - rewardPercentage.current
+    if (deltaPercentage > 0) {
+      const colorGreen = '#1bc47d'
+      return (
+        <div className="flex flex-row items-center">
+          <FaArrowUp style={{ color: colorGreen }} />
+          <div className={cn(`text-[${colorGreen}] text-sm`)}>+{deltaPercentage}</div>
+        </div>
+      )
+    }
+    if (deltaPercentage < 0) {
+      const colorRed = '#f14722'
+      return (
+        <div className="flex flex-row items-center">
+          <FaArrowDown style={{ color: colorRed }} />
+          <div className={cn(`text-[${colorRed}] text-sm`)}>{deltaPercentage}</div>
+        </div>
+      )
+    }
+    return null
+  }, [rewardPercentage])
+  return (
+    <TableCell className={cn(tableHeaders[1].className, 'border-b-0')}>
+      <div className="flex flex-row gap-x-1 font-rootstock-sans justify-center gap-2 ">
+        <div>{rewardPercentage?.current}</div>
+        {renderDelta}
+      </div>
+    </TableCell>
+  )
+}
+
+export const LastCycleRewardCell = ({ rewards }: { rewards: Reward[] }) => {
+  return (
+    <TableCell className={cn(tableHeaders[2].className, 'border-solid')}>
+      <LazyRewardCell rewards={rewards} />
+    </TableCell>
+  )
+}
+
+const ProjectedRewardCell = ({ rewards }: { rewards: Reward[] }) => {
+  return (
+    <TableCell className={cn(tableHeaders[3].className, 'border-solid')}>
+      <LazyRewardCell rewards={rewards} />
+    </TableCell>
+  )
+}
+
+type ShareProps = {
+  // a percentage without decimals
+  share: bigint
+}
+
+const ShareCell = ({ share }: ShareProps) => {
+  return (
+    <TableCell className={cn(tableHeaders[4].className, 'border-solid text-center border-b-0 items-center')}>
+      <div className="flex flex-row gap-2 items-center">
+        <ProgressBar progress={Number(share)} color="#1bc47d" />
+        <Label>{share.toString()}%</Label>
+      </div>
+    </TableCell>
+  )
+}
+const ActionCell = () => {
+  /* TODO: manage the button status 
+    - disabled when the backer cannot vote on the Builder
+    - variant=primary when the builder is selected and text changed to "Selected"
+    - variant=secondary by default and text is "Select"
+  */
+  /* TODO: add the onClick event
+   *  - it needs to interact with the allocation context to add the builder to the selected builders
+   */
+  return (
+    <TableCell className={cn(tableHeaders[5].className, 'border-solid align-center')}>
+      <Button variant="secondary" /* disabled={true} */>Select</Button>
+    </TableCell>
+  )
+}
+
+enum RewardsColumnKeyEnum {
+  builder = 'builder',
+  lastCycleReward = 'lastCycleReward',
+  projectedReward = 'projectedReward',
```

Comment:
I would keep the same name, 
```suggestion
  estimatedRewards = 'estimatedRewards',
```